src/nvim/auevents.lua

```diff
@@ -1,156 +1,139 @@
 return {
+  --- @type [string, string[], boolean][] List of [eventname, aliases, window-local event] tuples.
   events = {
```

Comment:
This annotation suggests that an event can have multiple aliases, but both `gen_events.lua` and `gen_vimvim.lua` only support a single alias per event. Perhaps you should add `ipairs()` to support multiple aliases?